e2e/vue3-vite-e2e/project.json

```diff
@@ -4,9 +4,9 @@
   "sourceRoot": "e2e/vue3-vite-e2e/src",
   "targets": {
     "e2e": {
-      "executor": "@nrwl/nx-plugin:e2e",
+      "executor": "@nrwl/jest:jest",
```

Comment:
Using the `jest` executor instead of the specific `e2e` one. The reason is that all `nx-plugin:e2e` does differently is build a target first. Since we do that already in CI anyway, it doesn't seem worth the extra wrapper around Jest. `@nrwl/jest:jest` seems to be updated more frequently, and it has more features.